packages/rightdown/src/processors/ast.ts

```
@@ -0,0 +1,171 @@
+import { remark } from 'remark';
+import { visit } from 'unist-util-visit';
+import type { Root, Code } from '../types/mdast.js';
```

ESLint

[error] 'Root' is defined but never used.

(@typescript-eslint/no-unused-vars)